src/thor/timedistancebssmatrix.cc

```diff
@@ -43,22 +43,6 @@ float TimeDistanceBSSMatrix::GetCostThreshold(const float max_matrix_distance) c
   return max_matrix_distance / (2.0f * kMPHtoMetersPerSec);
 }
 
-// Clear the temporary information generated during time + distance matrix
-// construction.
-void TimeDistanceBSSMatrix::clear() {
```

Comment:
moved to header and renamed to `reset` where it doesn't clear most destination info